src/components/QuickstartOverview.js

```diff
@@ -23,71 +22,50 @@ const allowedElements = [
 const QuickstartOverview = ({ quickstart }) => {
   return (
     <>
-      <h2> What&apos;s included </h2>
-      <div
-        css={css`
-          display: grid;
-          grid-gap: 1rem;
-          grid-template-columns: repeat(3, 1fr);
-
-          @media (max-width: 1180px) {
-            grid-template-columns: repeat(1, 1fr);
-          }
-        `}
-      >
-        {quickstart.dashboards.map((dashboard, index) => (
-          <OverviewTile
-            key={index}
-            title={dashboard.name}
-            image={dashboard.screenshots[0]}
-            description={dashboard.description}
-            tag="Dashboard"
-          />
-        ))}
-        {quickstart.alerts.map((alert, index) => (
-          <OverviewTile
-            key={index}
-            title={alert.name}
-            description={alert.details}
-            tag="Alert"
-          />
-        ))}
-        {quickstart.documentation.map((doc, index) => (
-          <OverviewTile
-            key={index}
-            title={doc.name}
-            description={doc.description}
-            tag="Doc"
-          />
-        ))}
-      </div>
       {quickstart.description && (
         <div
           css={css`
```

Comment:
To avoid having to list the font color for each html tag, we could put the color up here and let it cascade to the other styles.
```suggestion
          css={css`
            color: #1d252c;
```